### ISSUE TITLE
I/O

### ISSUE THREAD

**Issue by hi-bye125** [hi-bye125/ri-radio-scope#2]
Currently, this code works as a discord-controlled/website controlled bot. The output is a pdf file which is kinda hard to parse. No alt/az or eq control currently available pls collab w motors on this. 

make a csv/xml file for the output (can this be uploaded to drive automatically? (there is functionality for this somewhere in the code, but default is pdf so not great) or else we could try aws, but ideally we dont want to spend anything on storage at least) also any ideas for how to control? discord bot seems a bit funny

we can connect to PDLP@RI to allow it to upload data?

**Comment by D0m1n1c011** [comment]
Stupid question but do you have an example output for the pdf? If it's text you can probably use a PDF reader to read the text into a dataframe and convert it into csv then use Google Cloud (probably free) and an API key to send it to drive. If it's an image output, idk what to do

**Comment by hi-bye125** [comment]
@D0m1n1c011 good point actually we could use google cloud api to send to a google acc. then periodically download onto a drive or sth. the main problem w pdf is you cant really read the spectrograms and stuff like that so idt its a good idea to use pdf reader?

i mean theres a csv output we just have to figure out how to make it primary
